lib/user_agent_parser/device.rb

```diff
@@ -2,12 +2,14 @@
 
 module UserAgentParser
   class Device
+    DEFAULT_FAMILY = 'Other'
```

Comment:
Can we move this constant to UserAgentParser module level?

Reply:
It can potentially be different values for Device, OS and UserAgent

Reply:
That's what I thought. I was just thinking that we could use a more global now and change it once it happens. Let's leave as is.